### PR TITLE
Fix: add required rust version in cargo.toml

### PR DESCRIPTION
**Pull request by dp-0**
close #191

### CODE REVIEW

**Comment by odysa** [comment]
It closes #192

File: Cargo.toml [hunk 1]
```diff
@@ -26,6 +26,7 @@ homepage = "https://rust.iceberg.apache.org/"
 
 repository = "https://github.com/apache/iceberg-rust"
 license = "Apache-2.0"
+rust-version = "1.75.0"
```
Comment (Xuanwo) [inline Cargo.toml:29]: Please also update packages's `rust-version` to `workspakce = true`

Comment (dp-0) [reply Cargo.toml:29]: Hi there! Could you please provide more details or clarify where exactly you are suggesting to add this setting?

**Review by liurenjie1024 — APPROVED** [review]
LGTM, thanks!

**Review by Xuanwo — APPROVED** [review]
Thanks!

**Review by Fokko — APPROVED** [review]

**Pull request merged**